install_requires.txt

```diff
@@ -1,3 +1,4 @@
 Django>=1.7,<=1.9
 djangorestframework>=3.1.0,<=3.3.0
 inflection==0.3.1
+django-debug-toolbar==1.4
```

Comment:
Developer requirements should go into `requirements.txt`, which is only referenced when running the development server/tests. `install_requires.txt` requirements will be pulled into all projects that require DREST.